HashVerify/Verify-Hash.psm1

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
do we have any other module that could fit this cmdlet in to?

Reply:
Couldn't find another module similar enough to put this in, and I think this function could be independent enough to has its own module